WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3546,8 +3547,14 @@ public void onCapturePhotoClicked() {
     }
 
     @Override
-    public void onAddVideoClicked(boolean allowMultipleSelectio) {
-        onPhotoPickerIconClicked(PhotoPickerIcon.ANDROID_CHOOSE_VIDEO, allowMultipleSelectio);
+    public void onAddVideoClicked(boolean allowMultipleSelection) {
+        onPhotoPickerIconClicked(PhotoPickerIcon.ANDROID_CHOOSE_VIDEO, allowMultipleSelection);
+    }
+
+    @Override
+    public void onAddMediaClicked(boolean allowMultipleSelection) {
+        mAllowMultipleSelection = allowMultipleSelection;
+        WPMediaUtils.launchMediaLibrary(this, mAllowMultipleSelection);
```

Comment:
I'm definitely not familiar enough with Android and I'd recommend another reviewer, but I was aware of the [MediaBrowserType](https://github.com/wordpress-mobile/WordPress-Android/blob/00acdb8ee28c1221030eb45d5016398d8b8722cf/WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserType.java) enum, and I see that the other addMedia methods use `ActivityLauncher.viewMediaPickerForResult` instead. Is there a reason not to use the same thing?

Reply:
Great question! The `viewMediaPickerForResult` method actually prepares an intent with the "in-house" `MediaBrowserActivity` (i.e. "WordPress Media Library"). At first glance at the code, I had the same thought, and that may be an indication that the name is ambiguous. OTOH, I'm not sure I can offer a better name, since the `MediaBrowserActivity` itself has menu items that call through to the device picker (which, incidentally, use the `WPMediaUtils` methods).

Also, I will take your advice about adding an Android reviewer (more familiar with this area of the code base)  :+1:

Reply:
The "WordPress Media Library" flow has also been addressed in this PR.